packages/stencil/src/components/datacom-alert-banner/datacom-alert-banner.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+
+@Component({
+  tag: 'datacom-alert-banner',
+  styleUrl: 'datacom-alert-banner.css',
+  shadow: true,
+})
+export class DatacomAlertBanner {
+  @Prop() copy: string;
+  @Prop() link: string;
+  @Prop() url: string;
+
+  closeIcon = getSvg('close', { class: 'dc-close-icon' });
+
+  render() {
+    return (
+      <Host>
+        <div class="dc-alert-banner">
+          {this.copy}
+          <a href={this.url}>{this.link}</a>
+          {this.closeIcon}
```

Comment:
Shouldn't this close icon actually close the banner on click? You should probably look to add functionality to do that. Maybe wrap this close icon with a button element and then add a click handler on to that.

Reply:
Done